internal/metrics/metrics.go

```diff
@@ -87,33 +87,38 @@ func newStateCounter(ns *gometrics.Namespace, desc *prometheus.Desc) *StateCount
 }
 
 // Get returns the count of containers in running, paused, and stopped states
-func (ctr *StateCounter) Get() (running int, paused int, stopped int) {
+func (ctr *StateCounter) Get() (running, paused, stopped int) {
 	ctr.mu.RLock()
 	defer ctr.mu.RUnlock()
 
-	states := map[string]int{
-		"running": 0,
-		"paused":  0,
-		"stopped": 0,
-	}
+	running, paused, stopped = 0, 0, 0
 	for _, state := range ctr.states {
-		states[state]++
+		switch state {
+		case "running":
+			running++
+		case "paused":
+			paused++
+		case "stopped":
+			stopped++
+		}
 	}
-	return states["running"], states["paused"], states["stopped"]
+	return
```

Comment:
Can you change this to explicitly return the vars? (We try to avoid naked returns, and have a linter for that);

```
internal/metrics/metrics.go:105:2: naked return in func `Get` with 16 lines of code (nakedret)
	return
	^
```